Composer/packages/client/package.json

```
@@ -37,7 +37,7 @@
     "@uifabric/styling": "^7.10.4",
     "adaptive-expressions": "^4.8.0-preview-110700",
     "axios": "^0.18.0",
-    "botbuilder-lg": "^4.8.0-preview-109324",
+    "botbuilder-lg": "^4.9.0-preview-117748",
```

```suggestion
    "botbuilder-lg": "4.9.0-preview-117748",
```

Let's lock to this version.